hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/init/ExecutionPlan.java

```diff
@@ -344,7 +344,8 @@ private void initDag() {
                 .flatMap(List::stream)
                 .map(EdgeDef::partitioner)
                 .filter(Objects::nonNull)
-                .forEach(p -> p.init(partitionService::getPartitionId));
+                .forEach(partitioner ->
+                        partitioner.init(object -> partitionService.getPartitionId(serializationService.toData(object))));
```

Comment:
very minor but I find it more readable if the closing `)` is on the next line here.